win_build/unittests_vs2019.vcxproj

```diff
@@ -96,6 +96,7 @@
     <ClCompile Include="..\tests\unit-tests\lib\test_url.cpp" />
     <ClCompile Include="..\tests\unit-tests\lib\test_base64.cpp" />
     <ClCompile Include="..\tests\unit-tests\lib\test_util.cpp" />
+    <ClCompile Include="..\tests\unit-tests\lib\test_synch.cpp" />
```

Comment:
No need to put linux-only unit test to vcxproj file, please remove it from here

Reply:
Removed